### ISSUE TITLE
downloads on the main website don't work

### ISSUE THREAD

**Issue by irvnriir** [microsoft/vscode#193276]
Does this issue occur when all extensions are disabled?: Yes/No

- OS Version: Firefox

i also tried VPN

![image](https://github.com/microsoft/vscode/assets/13949704/6e976806-637e-41e1-b6ee-07add41370d1)

**Comment by rzhao271** [comment]
I'm unable to reproduce the issue.
@irvnriir does the issue occur for other downloads on the [Download page](https://code.visualstudio.com/download)?

**Comment by irvnriir** [comment]
@rzhao271 seems that with the most/all .

try blocking all telemetry and advertising junk . (monitor.azure.com and etc.)

im also getting these errors:
```
This page uses the non standard property “zoom”. Consider using calc() in the relevant property values, or using “transform” along with “transform-origin: 0 0”. (code.visualstudio.com/docs/?dv=win64)

ReferenceError: oneDS is not defined
    <anonymous> https://code.visualstudio.com/docs/?dv=win64:33
    
Content Security Policy: Ignoring ‘x-frame-options’ because of ‘frame-ancestors’ directive.

Blocked XHR POST https://marketplace.visualstudio.com/_apis/public/gallery/extensionquery
```

and sometimes video/player/amp related errors .

**Comment by rzhao271** [comment]
In your screenshot, it looks like https://az764295.vo.msecnd.net/stable/abd2f3db4bdb28f9e95536dfa84d8479f1eb312d/VSCode-win32-x64-1.82.2.zip doesn't work, but I'm unable to reproduce the issue.
Did you block that domain?

**Comment by irvnriir** [comment]
im only getting messages that Google Analytics and such items are blocked, nothing related to this domain or actual content in general . i also tried using this address directly .

**Comment by irvnriir** [comment]
i tried another VPN and it worked, apparently at least 1 of my and 1 of VPN provider's IPs are blocked on your side .

**Comment by VSCodeTriageBot** [comment]
We closed this issue because we [don't plan to address it](https://aka.ms/vscode-out-of-scope) in the foreseeable future. If you disagree and feel that this issue is crucial: we are happy to listen and to reconsider.

If you wonder what we are up to, please see our [roadmap](https://aka.ms/vscoderoadmap) and [issue reporting guidelines](https://aka.ms/vscodeissuereporting).

Thanks for your understanding, and happy coding!
